### PR TITLE
Papernach doc correction

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference : https://razorpay.com/docs/api/payments/recurring-payments/paper-nach

### CODE REVIEW

File: documents/papernach.md [hunk 1]
```diff
@@ -40,7 +41,25 @@ $api->order->create(array('amount' => 0,'currency' => 'INR','method' => 'nach','
...
 
-All parameters listed [here](https://razorpay.com/docs/api/route/#create-transfers-from-payments) are supported
+| Name            | Type    | Description                                                                  |
+|-----------------|---------|------------------------------------------------------------------------------|
+| amount*   | integer      | The amount to be captured (should be equal to the authorized amount, in paise) |
+| currency*   | string  | The currency of the payment (defaults to INR)  |
+| customerId*   | string      | The id of the customer to be fetched |
+| method*      | string  | Payment method used to make the registration transaction. Possible value is `nach`.  |
+| receipt      | string  | Your system order reference id.  |
+| token["auth_type"]  | string  | Possible value is `physical`|
```
Comment (sofiya02) [inline documents/papernach.md:51]: All parameters listed [here] are supported

File: documents/papernach.md [hunk 2]
```diff
@@ -40,7 +41,25 @@ $api->order->create(array('amount' => 0,'currency' => 'INR','method' => 'nach','
...
+| token["max_amount"]  | integer  | Use to set the maximum amount per debit request. The value can range from `500` - `1000000000` (1cr, default value)  |
+| token["expire_at"]  | integer | The timestamp, in Unix format, till when the  registration link should expire |
+| token["notes"]  | object  | A key-value pair  |
+| bank["account_number"]  | string  | Customer's bank account number.  |
```
Comment (sofiya02) [inline documents/papernach.md:55]: All parameters listed [here] are supported

File: documents/papernach.md [hunk 3]
```diff
@@ -116,7 +135,32 @@ $api->subscription->createSubscriptionRegistration(array('customer' => array('na
 ```
 
 **Parameters:**
-All parameters listed [here](https://razorpay.com/docs/api/recurring-payments/paper-nach/authorization-transaction/#121-create-a-registration-link) are supported
```
Comment (sofiya02) [inline documents/papernach.md:119]: change the parameter URL in readme  to - https://razorpay.com/docs/api/payments/recurring-payments/paper-nach/create-authorization-transaction/#112-create-an-order

File: documents/emandate.md [hunk 4]
```diff
@@ -36,19 +36,28 @@ $api->customer->create(array('name' => 'Razorpay User', 'email' => 'customer@raz
...
+| token["max_amount"]  | integer  | Use to set the maximum amount per debit request. The value can range from `500` - `1000000000` (1cr, default value)  |
+| token["expire_at"]  | integer | The timestamp, in Unix format, till when the  registration link should expire |
+| token["notes"]  | object  | A key-value pair  |
+| bank["account_number"]  | string  | Customer's bank account number.  |
+| bank["ifsc_code"]  | string  | Customer's bank IFSC  |
+| bank["beneficiary_name"]  | string  |  Customer's name  |
+| bank["account_type"]  | string  | Customer's bank account. Possible value is `saving`(default), `current`  |
+| notes | object  | A key-value pair  |
 
 **Response:**
```
Comment (sofiya02) [inline documents/emandate.md:62]: check in view state. **Response** is inside parameter table

File: documents/emandate.md [hunk 5]
```diff
@@ -63,20 +72,33 @@ Please refer this [doc](https://razorpay.com/docs/api/recurring-payments/emandat
...
 | description*    | string  | A description that appears on the hosted page. For example, `12:30 p.m. Thali meals (Gaurav Kumar`).                                                             |
-| subscription_registration           | array  | Details of the authorization payment.                      |
-| notes           | array  | A key-value pair                                                             |
+| subscription_registration["method"]           | string  | The authorization method. In this case, it will be `emandate`.|
+| subscription_registration["auth_type"]           | string  | Possible values `netbanking`, `debitcard` or `aadhaar`|
+| subscription_registration["max_amount"]           | integer  | The maximum amount, in paise, that a customer can be charged in one transaction.|
+| subscription_registration["expire_at"]           | integer  | when you can use the token (authorization on the payment method) to charge the customer subsequent payments. Defaults to 10 years for `emandate`. |
+| subscription_registration["bank"]["account_number"]  | integer  | Customer's bank account number.  |
```
Comment (sofiya02) [inline documents/emandate.md:93]: All parameters listed [here] are supported

File: documents/papernach.md [hunk 6]
```diff
@@ -40,7 +41,25 @@ $api->order->create(array('amount' => 0,'currency' => 'INR','method' => 'nach','
 
 **Parameters:**
 
-All parameters listed [here](https://razorpay.com/docs/api/route/#create-transfers-from-payments) are supported
```
Comment (sofiya02) [inline documents/papernach.md:43]: https://razorpay.com/docs/api/payments/recurring-payments/paper-nach/create-authorization-transaction/#112-create-an-order

**Review by sofiya02 — CHANGES_REQUESTED** [review]

File: documents/emandate.md (hunk 5 above)
Comment (ankitdas13) [reply documents/emandate.md:93]: add param link

File: documents/papernach.md (hunk 3 above)
Comment (sofiya02) [reply documents/papernach.md:119]: please update

**Review by sofiya02 — CHANGES_REQUESTED** [review]

File: documents/papernach.md (hunk 6 above)
Comment (ankitdas13) [reply documents/papernach.md:43]: its there but its not showing check this
https://github.com/razorpay/razorpay-php/blob/papernach_doc_correction/documents/papernach.md?plain=1#L51

File: documents/papernach.md (hunk 3 above)
Comment (ankitdas13) [reply documents/papernach.md:119]: added all param link for `customer` & `subscription_registration` but not showing. can you check this link
https://github.com/razorpay/razorpay-php/blob/papernach_doc_correction/documents/papernach.md?plain=1#L136

**Pull request merged**